src/components/ui/meetingDialog.tsx

```diff
@@ -0,0 +1,205 @@
+'use client';
+
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { useState } from "react";
+import dayjs from "dayjs";
+import { Plus } from "lucide-react";
+
+interface MeetingDialogProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+export function MeetingDialog({ isOpen, onClose }: MeetingDialogProps) {
+  const [summary, setSummary] = useState("");
+  const [description, setDescription] = useState("");
+  const [startDateTime, setStartDateTime] = useState<string>(dayjs().add(1, "day").format('YYYY-MM-DD'));
+  const [endDateTime, setEndDateTime] = useState<string>(dayjs().add(1, "day").add(1, "hour").format('YYYY-MM-DD'));
+  const [attendees, setAttendees] = useState<string[]>([""]);
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    const meetingData = {
+      summary,
+      description,
+      start: {
+        dateTime: dayjs(startDateTime).toISOString(),
+        timeZone: "Asia/Kolkata",
```

Comment:
Why are we limited to Asia/Kolkata, I don't think we should limit timezone, 
@arpit2222 please share your input